docs/reference/cat.asciidoc

```diff
@@ -95,6 +95,12 @@ green wiki1 3 0 10000 413 103776272 103776272
 green foo   1 0   227   0   2065131   2065131
 --------------------------------------------------
 
+If you want to change the <<time-units,time units>>, use `time` parameter.
+
+If you want to change the <<size-units,size units>>, use `size` parameter.
+
+If you want to change the <<size-units,byte units>>, use `bytes` parameter.
```

Comment:
Yes. When I will add it! :p See https://github.com/elastic/elasticsearch/pull/17779/files/8d05b705b887578c59909bacfcd7509fbd191a21#r59859803